main.css

```diff
@@ -0,0 +1,500 @@
+  font-weight: normal;
+  font-size: 18px;
+  color: #272a31;
+  margin: 0;
+}
+
+.speaker-about {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  color: #ec5242;
+  margin-bottom: 0;
+  font-style: italic;
+  text-transform: lowercase;
+}
+
+.speaker-about::after {
+  display: block;
+  width: 15px;
+  height: 2px;
+  background-color: #d3d3d3;
+  content: '';
+  margin: 5px 0;
+}
+
+.talk-about {
+  margin: 0;
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  font-size: 14px;
+  color: #272a31;
+}
+
+.partner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #272a31;
+  background-image: url(background2.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.partner-title {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  font-size: 20px;
+  color: #fff;
+}
+
+.partners-container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 30px 0;
+}
+
+.partner-img {
+  width: 100px;
+  height: 100px;
+  margin: 0 15px;
+}
+
+footer {
+  display: flex;
+  justify-content: space-around;
+}
+
+.logo {
+  text-decoration-line: none;
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  font-size: 20px;
+  color: #ec5242;
+  padding: 10px 10px;
+  background-color: #272a31;
+  text-align: center;
+  border-radius: 4px;
+  margin: 30px 10px;
+  line-height: 30px;
+}
+
+.copy-right {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  font-size: 14px;
+  margin: 50px 0;
+  text-align: center;
+}
+
+@media (max-width: 766px) {
+  nav .logo {
+    display: none;
+  }
+
+  .user-bar {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .hamb-icon {
+    display: none;
+  }
+
+  .user-bar {
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    background-color: #272a31;
+    height: 35px;
+  }
+
+  .user-bar-link {
+    text-decoration: none;
+    font-family: 'lato', sans-serif;
+    font-weight: normal;
+    font-size: 17px;
+    color: #fff;
+    padding: 0 20px;
+    cursor: pointer;
+  }
+
+  .user-bar-smedia {
+    width: 25px;
+    height: 25px;
+    cursor: pointer;
+    padding: 0 20px;
+  }
+
+  nav {
+    background-color: #fff;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    height: 95px;
+  }
+
+  .menu-list.close {
+    display: none;
+  }
+
+  .menu-links {
+    display: flex;
+    position: static;
+    box-shadow: none;
+    border-radius: 0;
+    flex-direction: row;
+    justify-content: space-around;
+    padding: 0;
+  }
+
+  .menu-link {
+    font-size: 13px;
+    color: #535353;
+    padding: 20px 15px;
+    margin: 0 15px;
+  }
+
+  .menu-link:hover {
+    color: #ec5242;
+  }
+
+  .menu-list.homepage {
+    display: none;
+  }
+
+  .main-title::before {
+    font-size: 35px;
+  }
+
+  .main-title {
+    font-size: 75px;
+    margin-left: 13%;
+  }
+
+  .headline-description {
+    font-size: 35px;
```

Comment:
- [ ] To match the original design better, you should consider making this a bit smaller.

Behance design:
![image](https://user-images.githubusercontent.com/66967274/142277824-6632457f-cfbc-43b9-8ecd-b63d06ab3728.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/142277763-5e0c08fc-2288-41ae-a7b0-aff838d0735b.png)